src/jsMain/kotlin/io/beatmaps/util/audio.kt

```diff
@@ -8,10 +8,10 @@ fun useAudio() = useRef(
     Audio().also {
         it.volume = 0.4
     }
-).also { audio ->
+).also {
     useEffectOnce {
         cleanup {
-            audio.current?.pause()
+            it.current?.pause()
```

Comment:
Any reason?

Reply:
Moved it to a file where it shadowed another variable. Then moved it back and forgot to change it. I'll revert that.